wallet/src/main/scala/org/bitcoins/wallet/internal/UtxoHandling.scala

```diff
@@ -39,6 +39,44 @@ private[wallet] trait UtxoHandling extends WalletLogger {
     spendingInfoDAO.findAllUnspentForAccount(hdAccount)
   }
 
+  protected def updateUtxoConfirmedState(
+      txo: SpendingInfoDb,
+      blockHash: DoubleSha256DigestBE): Future[SpendingInfoDb] = {
+    for {
+      confsOpt <- chainQueryApi.getNumberOfConfirmations(blockHash)
```

Comment:
It looks very similar to the change in TransactionProcessing.scala. DRY it up, please.